xapian-applications/omega/templates/query

```diff
@@ -37,87 +37,96 @@ $if{$opt{topterms},$include{inc/toptermsjs}}
- </div>
+ $map{$topterms,<span class="query-suggested-element" style="white-space:nowrap"><input type=checkbox name=X value="$html{$prettyterm{$_}}" onClick="C(this)">$html{$prettyterm{$_}}</span> }
+ <noscript><input type=hidden name=ADD value=1></noscript>
+ </fieldset>
 }
-<br>
-$if{$suggestion,Did you mean: <a href="$html{$or{$env{SCRIPT_NAME},omega}?DB=$url{$dbname}&P=$url{$suggestion}$if{$ne{$hitsperpage,10},&HITSPERPAGE=$hitsperpage}$if{$fmt,&FMT=$url{$fmt}}$if{$cgi{COLLAPSE},&COLLAPSE=$url{$cgi{COLLAPSE}}}$map{$relevants,&R=$_}$if{$cgi{THRESHOLD},&THRESHOLD=$url{$cgi{THRESHOLD}}}$map{$cgilist{B},&B=$url{$_}}&SPELL=0}">$html{$suggestion}</a><br>}
+$if{$suggestion,Did you mean: <a class="query-suggestion" href="$html{$or{$env{SCRIPT_NAME},omega}?DB=$url{$dbname}&P=$url{$suggestion}$if{$ne{$hitsperpage,10},&HITSPERPAGE=$hitsperpage}$if{$fmt,&FMT=$url{$fmt}}$if{$cgi{COLLAPSE},&COLLAPSE=$url{$cgi{COLLAPSE}}}$map{$relevants,&R=$_}$if{$cgi{THRESHOLD},&THRESHOLD=$url{$cgi{THRESHOLD}}}$map{$cgilist{B},&B=$url{$_}}&SPELL=0}">$html{$suggestion}</a>}
+${Query metadata - shows query performance or document size}
 $or{$html{$error},
+<p class="query-metadata">
 $if{$eq{$msize,0},
 $if{$query,No documents match your query,
-<hr>Searching $nice{$dbsize} documents
-},
+Searching $nice{$dbsize} documents
+}</p>,
 $if{$not{$msizeexact},
  $nice{$add{$topdoc,1}}-$nice{$last} of about $nice{$msize} matches,
  $if{$and{$eq{$last,$msize},$eq{$topdoc,0}},
   All $nice{$msize} matches,
   $nice{$add{$topdoc,1}}$if{$ne{$add{$topdoc,1},$last},-$nice{$last}} of exactly $nice{$msize} matches}
 }
-<hr>
-</center>
-$list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
-<br><small>Search took $time seconds</small>
+<br/>
+$list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},,,}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
+Search took $time seconds
+</p>
+${Results}
 <table>
-$hitlist{<tr><td valign=top>
+$hitlist{<tr><td class="result-metadata">
 ${<img src="/icons/omega/score-$div{$percentage,10}.png" alt="$percentage%" height=16 width=32>}
-<div title="$percentage%" style='float:left;width:60px;height:10px;border:1px solid black;margin-top:4px;'>
-<div style='width:$div{$mul{$percentage,6},10}px; height:10px; background-color: red;'>
+<div class="result-metadata-progress-container" title="$percentage%" >
+<div class="result-metadata-progress" style='width:$div{$mul{$percentage,6},10}px;'>
 </div></div>
-<div style='float:left;margin-top:2px;font-size:x-small;'>
+<div class="result-metadata">
 $if{$and{$field{modtime},$ne{$field{modtime},-1}},
 <script type="text/javascript">write_date($add{$field{modtime}});</script>
 <noscript>
-<span title="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
-Modified:<br><b>$html{$date{$field{modtime},%Y-%m-%d}}</b></span>
-</noscript><br>
+<span>Modified:<br/><time datetime="$html{$date{$field{modtime},%Y-%m-%d %H:%M:%S}}">
```

Comment:
In this, and other situations where you're introduced `datetime`, you must:

 * provide the `datetime` attribute in the [correct format](https://www.w3.org/TR/html51/infrastructure.html#dates-and-times), which is for computers
 * retain the `title` on the container, since that's for humans

Again, you can avoid `br` elements here.